sanic_ext/extensions/openapi/ui/swagger.html

```diff
@@ -5,7 +5,7 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="stylesheet" type="text/css"
-            href="//cdnjs.cloudflare.com/ajax/libs/swagger-ui/__VERSION__/swagger-ui.min.css">
+            href="https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/__VERSION__/swagger-ui.min.css">
```

Comment:
Why?

Reply:
It resolved a warning on my text editor. Also, it can make it secure by explicitly use HTTPS?